### ISSUE TITLE
How do you remap mouse movement to nothing?

### ISSUE THREAD

**Issue by LargeSweater** [evilC/UCR#238]
Sometimes I want to disable my mouse input (while still plugged in) while playing a game that has DirectInput. Can UCR intercept and block all mouse input? If so, how? Thank you for your software!

Possibly related issue I created evilC/AutoHotInterception#81
https://github.com/Snoothy/UCR/issues/265

**Comment by evilC** [comment]
Yes, just make a profile with only inputs, and no outputs (And enable
blocking).
However, unless you have a 2nd mouse, then you would not want to do this,
as you could not then turn the profile off (Because the stop profile button
needs to be clicked with the mouse)
